torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -10043,9 +9999,8 @@ def forward(self, x):
                         loss = a.sum() + b.sum()
                         loss.backward()
 
-                    ws = dist.get_world_size()
                     for p in local_model.parameters():
-                        p.grad.data = p.grad / dist.get_world_size()
+                        p.grad.data = p.grad / 2
```

Comment:
This looks suspicious, is there a reason this should be hardcoded?

Reply:
I thought here we should do / 2 because the forward + backward is run twice, which is unrelated to world_size.

Setting to / 2 and run locally the test passed.

Reply:
I see, makes sense, maybe it's clearer to make the iteration count a constant and share it with the previous loop?